docs/webhooks.rst

```diff
@@ -93,6 +93,32 @@ GitLab
 * Leave the default **Push events** selected and mark **Tag push events** also
 * Finish by clicking **Add Webhook**
 
+.. _webhook-integration-gitea:
```

Comment:
We use `autosectionlabel`, this isn't needed.

Reply:
Ok, I just copied from the other sections 😬